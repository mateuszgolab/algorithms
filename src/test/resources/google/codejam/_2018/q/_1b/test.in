8
13 1
2
13 1
1
7 3
1 1 1
7 4
1 1 1 1
7 2
2 1
6 2
3 2
25 2
1 1
2 1
1