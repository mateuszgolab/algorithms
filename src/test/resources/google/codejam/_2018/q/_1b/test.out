Case #1: 103
Case #2: 104
Case #3: 101
Case #4: 101
Case #5: 101
Case #6: 100
Case #7: 100
Case #8: 100
